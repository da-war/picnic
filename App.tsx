import {StyleSheet, Text, View} from 'react-native';
import React from 'react';
import AppNavigator from './src/navigation/AppNavigator';

const App = () => {
  // return <AppNavigator />;
  return (
    <View>
      <Text>Hello</Text>
    </View>
  );
};

export default App;

const styles = StyleSheet.create({});
