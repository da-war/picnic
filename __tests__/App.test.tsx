import React from 'react';
import App from '../App';
import {render} from '@testing-library/react-native';

test('SnapShot', () => {
  const {toJSON} = render(<App />);
  expect(toJSON()).toMatchSnapshot();
});
