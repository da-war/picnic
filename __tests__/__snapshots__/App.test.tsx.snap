// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapShot 1`] = `
<View>
  <Text>
    Hello
  </Text>
</View>
`;
